usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -1431,6 +1436,18 @@ CK_RV token_specific_set_attrs_for_new_object(STDLL_TokData_t *tokdata,
 
     return ret;
 }
+#else
```

Comment:
The ifdef of this else excludes a bit too much. Function `token_specific_set_attrs_for_new_object()` must stay there, it not handles PKEY stuff, but also other stuff (e.g. FORCE_SENSITIVE handling). You should ifdef out only the PKEY related stuff from `token_specific_set_attrs_for_new_object()` but keep the rest.
Also make sure that you don't exclude function `token_specific_set_attrs_for_new_object` in `tok_struct.h`.